decidim-core/app/views/decidim/pages/_standalone.html.erb

```diff
@@ -1,21 +1,17 @@
-<div class="wrapper">
-  <div class="row column">
-    <div class="page-title-wrapper">
-      <h1 class="heading1 page-title">
-        <%= title %>
-      </h1>
-    </div>
+<%= render layout:"layouts/decidim/shared/layout_center" do %>
 
-    <%= cell "decidim/tos_page", :announcement %>
+  <div class="text-center py-12">
+    <h1 class="h1 decorator inline-block text-left">
+      <%= title %>
+    </h1>
+  </div>
 
-    <div class="columns small-12">
-      <div class="card">
-        <div class="card__content"><%= decidim_sanitize_editor translated_attribute page.content %></div>
-      </div>
-    </div>
+  <%= cell "decidim/tos_page", :announcement %>
 
-    <div class="columns small-12">
-      <%= cell "decidim/tos_page", :form %>
-    </div>
+  <div class="editor-content">
```

Comment:
The standalone page looks a bit cluttered on mobile right now, maybe we should add a bit of spacing around the text?

![Cluttered standalone page](https://user-images.githubusercontent.com/864340/184160213-fe8c89f8-9486-4e25-8a95-e2c62f984473.png)